### PR TITLE
transpower: set LOCAL_SDK_VERSION to current

### PR DESCRIPTION
**Pull request by alviteri**
to avoid build problems in android pie

Signed-off-by: David Viteri <davidteri91@gmail.com>

### CODE REVIEW

**Comment by MarijnS95** [comment]
I tried this locally and it seems to need `LOCAL_PRIVATE_PLATFORM_APIS := true` instead, because one of the dependencies (`telephony-common`, depended on by `libpower`) uses private APIs.

Related:
https://android-review.googlesource.com/c/platform/frameworks/opt/telephony/+/624469

**Comment by alviteri** [comment]
yeah marin ..... I have the change locally ... let me update .... thanks

**Comment by alviteri** [comment]
updated and thanks for comment ....

**Comment by MarijnS95** [comment]
And thank you for upgrading sony AOSP to Pie :+1:

File: TransPowerAcc/Android.mk [hunk 1]
```diff
@@ -6,6 +6,9 @@ include $(CLEAR_VARS)
 LOCAL_PACKAGE_NAME := TransPowerAcc
 LOCAL_CERTIFICATE := platform
 LOCAL_PRIVILEGED_MODULE := true
+ifeq (1,$(filter 1,$(shell echo "$$(( $(PLATFORM_SDK_VERSION) >= 28 ))" )))
```
Comment (jzoran) [inline TransPowerAcc/Android.mk:9]: Shouldn't really use shell in *.mk files when there are alternatives.

**Comment by jzoran** [comment]
Thanks for your patch! I have a very similar patch that fixes this. When I have it tested thoroughly, I'll upload.

**Pull request closed without merging**